.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
```

Comment:
is github showing a status if the workflow action failed / passed?

Reply:
I added badges to the readme file so the current status is shown in github and pypi (they are showing the readme file as documentation)